deploy/mayastor-daemonset.yaml

```diff
@@ -65,8 +64,8 @@ spec:
             cpu: "1"
             memory: "500Mi"
             hugepages-2Mi: "1Gi"
-      - name: mayastor-grpc
-        image: mayadata/mayastor-grpc:v0.2.0
+      - name: mayastor-csi
+        image: mayadata/mayastor-csi:v0.2.0
```

Comment:
I would like to have "latest" in develop and "0.2.0" in master, but that's slightly more complicated with gitflow model when the changes are simply merged from develop to release to master branch (ideally). So keeping it as it is until we decide otherwise.